sources/packages/api/src/utilities/address-utils.ts

```diff
@@ -0,0 +1,24 @@
+import { AddressDetailsModel } from "../services";
+import { AddressInfo } from "../types";
+
+/**
+ * Util to transform address details.
+ * @param addressDetails is a partial AddressDetailsModel.
+ * @returns address info as AddressInfo.
+ */
+export function transformAddressDetails(
+  addressDetails: Partial<AddressDetailsModel>,
```

Comment:
Will all the attributes coming from addressDetails be optional ?

Reply:
good point. will fix this